ui/templates/robots.txt

```diff
@@ -0,0 +1,5 @@
+User-agent: *
+Disallow: /dashboard/
+Disallow: /learner/
+Disallow: /learners/
+Disallow: /api/
```

Comment:
Makes sense to me. My only concern is whether the program pages (i.e. /dedp/ and /scm/) have a dependency on /api even for not logged in users.